### ISSUE TITLE
Responsive Font Size Accessibility

### ISSUE THREAD

**Issue by irmerk** [accordproject/cicero-ui#58]
# WCAG 2.0 Level AA [1.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html) (Resize Text)

## Tasks
- All fonts should be sized relatively, which will be more responsive and allow better accessibility to those who magnify or zoom on their devices. Move away from `px` and instead favor `rem`, `em`, or `%`.

## Related Issues: 
- Issue [#43](https://github.com/accordproject/template-studio-v2/issues/43) in Template Studio v2

**Comment by irmerk** [comment]
### Hacktoberfest:
This issue can possibly provide for multiple pull requests.

**Comment by florenciasilva** [comment]
Hi, again! I would love to work on this to continue what was done in [#89 of template-studio-v2](https://github.com/accordproject/template-studio-v2/pull/89)

**Comment by irmerk** [comment]
@florenciasilva do you feel like #144 closes this issue for now?

**Comment by florenciasilva** [comment]
> @florenciasilva do you feel like #144 closes this issue for now?

Yes!

**Comment by irmerk** [comment]
🥳 Thank you!
